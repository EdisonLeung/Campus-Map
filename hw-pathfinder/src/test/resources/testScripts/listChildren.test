# Tests printing out the children to see if it prints the edges
# from least cost to most cost

# Create Graph
CreateGraph listChildren

# Add nodes
AddNode listChildren a
AddNode listChildren b
AddNode listChildren c

# Add edges
AddEdge listChildren a b 4.0
AddEdge listChildren a b 3.0
AddEdge listChildren a c 6.0
AddEdge listChildren a c 1.0

# print children
ListChildren listChildren a