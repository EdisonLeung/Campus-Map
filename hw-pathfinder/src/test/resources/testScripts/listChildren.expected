# Tests printing out the children to see if it prints the edges
# from least cost to most cost

# Create Graph
created graph listChildren

# Add nodes
added node a to listChildren
added node b to listChildren
added node c to listChildren

# Add edges
added edge 4.000 from a to b in listChildren
added edge 3.000 from a to b in listChildren
added edge 6.000 from a to c in listChildren
added edge 1.000 from a to c in listChildren

# print children
the children of a in listChildren are: b(3.000) b(4.000) c(1.000) c(6.000)